docs/release_notes/v0.8.0.md

```diff
@@ -0,0 +1,294 @@
+
+> **Note: This release contains a few [breaking changes](#breaking-changes).**
+
+See [this](#upgrading-to-dapr-0.8.0) section on upgrading Dapr to version 0.8.0.
+
+## New in this release
+
+### Dapr Runtime
+
+**Added** custom routes for pub/sub (#breaking-changes)
+
+**Enriched** http and grpc status code conversion.
+
+**Added** Healthz server for control plane.
+
+**Refactored** gRPC service and protobuf message definitions. (#breaking-changes)
+
+**Fixed** Actor activation and method invocation logic to handle app restarts.
+
+**Fixed** Unpopulated metadata for input bindings for HTTP app channel.
+
+**Improved** client trace span generation calls.
+
+**Added** liveness config.
+
+**Added** disable Span Generations when tracing rate is set to 0.
+
+**Added** trace headers in API response, which allows users to correlate with
+
+**Enhanced** mDNS to support hostname and IPs. (Thanks @jjcollinge)
+
+**Added** mqtt pubsub provider to daprd. (Thanks @kumaya)
+
+**Added** Pulsar pubsub options. (Thanks @elyst)
+
+**Added** API Authentication.
+
+**Added** bi-directional bindings support. (#breaking-changes)
+
+**Added** Set Metadata option to state store Get API.
+
+**Added** opentelemetry sementic convention.
+
+**Added** snssqs pubsub provider.
+
+**Added** Context timeout to internal grpc dial.
+
+**Added** Linear interval retry for Internal gRPC client.
+
+**Added** pending actor locks metric.
+
+**Generated** low-cardinality metric method label.
+
+**Fixed** marshalling error for gRPC binary metadata.
+
+**Improved** component folder search behavior by looking at home directory. (#breaking-changes)
+
+**Fixed** log options for contrib logger and enable metrics only for middleware. (Thanks @ksashikumar)
+
+**Fixed** http method label for HTTP appchannel metric.
+
+
+### Components
+
+**Added** AWS SNS/SQS pubsub. (Thanks @khous)
+
+**Added** Apache Pulsar. (Thanks @elyst)
+
+**Added** MQTT pubsub. (Thanks @kumaya)
+
+**Fixed** Azure Service Bus pubsub throttling. (Thanks @jjcollinge)
+
+**Fixed** SQL server state store integration tests. (Thanks @brooke-hamilton)
+
+**Updated** mDNS resolver to respect service IP address. (Thanks @jjcollinge)
+
+**Fixed** nethttpadaper to handle UserValue in request context. (Thanks @jjcollinge)
+
+**Improved** AWS authentication for dynamodb, kinesis, s3, sns, sqs, and secretmanager. (Thanks @trondhindenes)
+
+**Improved** dynamodb component to use BatchWriteItem for BulkSet and BulkDelete. (Thanks @ksashikumar)
+
+**Added** bi-directional bindings support for all bindings components.
+
+
+### CLI
+
+**Added** new flag to provide custom component directory path for dapr run. (Thanks @ksashikumar)
+
+**Added** verbose logs on failure of container uninstall and check if docker is installed. (Thanks @ksashikumar)
+
+**Improved** stop command to stop multiple apps. (Thanks @chinzhiweiblank)
+
+**Used** common components directory in user home directory.
```

Comment:
`Added default components directory in user home directory`

Reply:
Feel free to change this. You can edit this file using github ui :)